docs/sdk/dev/troubleshooting.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Troubleshooting
+description: Troubleshooting for the Spryker SDK document.
+template: concept-topic-template
+---
+# Troubleshooting
```

Comment:
Please adjust this file according to the Troubleshooting file template: https://github.com/spryker/spryker-docs/blob/master/_templates/troubleshooting-guide-template.md You can check the examples here:
- https://docs.spryker.com/docs/scos/dev/troubleshooting/troubleshooting.html
- https://docs.spryker.com/docs/cloud/dev/spryker-cloud-commerce-os/troubleshooting/troubleshooting-deployment-issues/troubleshooting-deployment-issues.html